packages/particles/src/ParticleBuffer.js

```diff
@@ -182,22 +184,24 @@ export default class ParticleBuffer
 
             if (property.unsignedByte)
             {
-                this.vao.addAttribute(
+                geometry.addAttribute(
+                    property.attributeName,
                     this.staticBuffer,
-                    property.attribute,
-                    gl.UNSIGNED_BYTE,
+                    0,
                     true,
+                    TYPES.UNSIGNED_BYTE,
                     this.staticStride * 4,
                     property.offset * 4
                 );
             }
             else
             {
```

Comment:
Maybe for later, but we could replace ```property.unsignedByte``` and add ```normalise``` and ```type```  to the properties instead. This is possible as we now have access to webGL types all the time. Then we could remove the if statement here

```
if (property.unsignedByte)
            {
                geometry.addAttribute(
                    property.attributeName,
                    this.staticBuffer,
                    0,
                    true,
                    TYPES.UNSIGNED_BYTE,
                    this.staticStride * 4,
                    property.offset * 4
                );
            }
            else
            {
                geometry.addAttribute(
                    property.attributeName,
                    this.staticBuffer,
                    0,
                    false,
                    TYPES.FLOAT,
                    this.staticStride * 4,
                    property.offset * 4
                );
            }
```
could become
```
                geometry.addAttribute(
                    property.attributeName,
                    this.staticBuffer,
                    0,
                    property.normalised,
                    property.type,
                    this.staticStride * 4,
                    property.offset * 4
                );
```

Same applies to the dynamic attribute adding too.